zcash_client_sqlite/src/lib.rs

```diff
@@ -685,17 +681,15 @@ impl<C: Borrow<rusqlite::Connection>, P: consensus::Parameters, CL, R> WalletRea
 
     fn get_derived_account(
         &self,
-        seed: &SeedFingerprint,
-        account_id: zip32::AccountId,
-        #[cfg(feature = "zcashd-compat")] legacy_address_index: Option<zcashd::LegacyAddressIndex>,
+        derivation: &Zip32Derivation,
```

Comment:
This change should occur instead of `legacy_address_index` addition in the previous commit.